css/w3develops.css

```diff
@@ -16,8 +16,14 @@ iframe#googleForm {
     width:100%;
 }
 
-.dropdown-list.dropdown-list-item:hover,
-.dropdown-list.dropdown-list-item:focus,
-.dropdown-listn.dropdow-list-item:active {
+.dropdow-list.dropdow-list-item:hover {
```

Comment:
Incorrect css selector.  For example one of the items in the https://github.com/w3develops/w3develops/blob/gh-pages/navigation.html#L18 has `<a class="dropdown-item dropdow-list-item" href="https://discord.gg/ckQ52gA" target="_blank" rel="noopener noreferrer">Discord Chat!!!</a>` You'll notice here that the `class` is set to `dropdown-item` and `dropdow-list-item` your css selector needs to target this so it should say `.dropdown-item.dropdow-list-item`... for its css selector.  If you've tested this, I'd be surprised if it worked.